dts/vendor/nordic/nrf7120_enga.dtsi

```diff
@@ -0,0 +1,957 @@
+				interrupts = <260 NRF_DEFAULT_IRQ_PRIORITY>;
+				status = "disabled";
+				endtx-stoptx-supported;
+				frame-timeout-supported;
+			};
+
+			comp: comp@106000 {
+				compatible = "nordic,nrf-comp";
+				reg = <0x106000 0x1000>;
+				interrupts = <262 NRF_DEFAULT_IRQ_PRIORITY>;
+				#io-channels-cells = <1>;
+				status = "disabled";
+			};
+
+			wdt31: wdt@109000 {
+				compatible = "nordic,nrf-wdt";
+				reg = <0x109000 0x1000>;
+				interrupts = <265 NRF_DEFAULT_IRQ_PRIORITY>;
+				status = "disabled";
+			};
+
+			gpio0: gpio@10a000 {
+				compatible = "nordic,nrf-gpio";
+				reg = <0x10a000 0x200>;
+				gpio-controller;
+				#gpio-cells = <2>;
+				port = <0>;
+				ngpios = <13>;
+				gpiote-instance = <&gpiote30>;
+				status = "disabled";
+			};
+
+			gpiote30: gpiote@10c000 {
+				compatible = "nordic,nrf-gpiote";
+				reg = <0x10c000 0x1000>;
+				instance = <30>;
+				status = "disabled";
+			};
+
+			clock: clock@10e000 {
+				compatible = "nordic,nrf-clock";
+				reg = <0x10e000 0x1000>;
+				interrupts = <270 NRF_DEFAULT_IRQ_PRIORITY>;
+				status = "disabled";
+			};
+
+			audio_auxpll: auxpll@130000 {
+				compatible = "nordic,nrf-auxpll";
+				reg = <0x130000 0x1000>;
+				interrupts = <304 NRF_DEFAULT_IRQ_PRIORITY>;
+				clocks = <&pclk32m>;
+				#clock-cells = <0>;
+				/*
+				 * Temporarily reading FICR addr 0 as FICR offsets
+				 * not defined yet
+				 */
+				nordic,ficrs = <&ficr 0>;
+				nordic,out-div = <12>;
+				nordic,out-drive = <0>;
+				nordic,current-tune = <9>;
+				nordic,sdm-disable;
+				nordic,range = "mid";
+				status = "disabled";
+			};
+
+			cpuapp_ppb: cpuapp-ppb-bus {
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				cpuapp_systick: timer@e000e010 {
+					compatible = "arm,armv8m-systick";
+					reg = <0xe000e010 0x10>;
+					status = "disabled";
+				};
+
+				cpuapp_nvic: interrupt-controller@e000e100  {
+					#address-cells = <1>;
+					compatible = "arm,v8m-nvic";
+					reg = <0xe000e100 0xc00>;
+					arm,num-irq-priority-bits = <3>;
+					interrupt-controller;
+					#interrupt-cells = <2>;
+				};
+			};
+		};
+
+		wifi_bellboard: mailbox@40074000{
+			reg = <0x40074000 0x1000>;
+			status = "disabled";
+			#mbox-cells = <1>;
+		};
+
+		cpuapp_bellboard: mailbox@40078000{
+			reg = <0x40078000 0x1000>;
+			status = "disabled";
+			#mbox-cells = <1>;
+		};
+
+		mram_controller: mram-controller@5004e000 {
+			compatible = "nordic,nrf-mramc";
+			reg = <0x5004e000 0x1000>;
+			#address-cells = <1>;
+			#size-cells = <1>;
+			interrupts = <78 NRF_DEFAULT_IRQ_PRIORITY>;
+
+			cpuapp_mram: mram@0 {
+				compatible = "soc-nv-flash";
+				reg = <0 DT_SIZE_K(3972)>;
+				erase-block-size = <4096>;
+				write-block-size = <4>;
+			};
+
+			cpuflpr_mram: mram@3e1000 {
+				compatible = "soc-nv-flash";
+				reg = <0x3e1000 DT_SIZE_K(116)>;
+				erase-block-size = <4096>;
+				write-block-size = <4>;
+			};
+		};
+	};
+
+	wifi: wifi {
+		compatible = "nordic,nrf7120";
```

Comment:
Maybe this needs some other compatibile? This one looks like something for the whole SoC and not just its Wi-Fi radio peripheral. Sorry if I'm wrong, I'm not a DT maintainer

```suggestion
		compatible = "nordic,nrf7120-wifi-radio";
```